package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "featureserver",
-  "version": "2.4.6",
+  "version": "2.5.0",
```

Comment:
no need to apply this change

Reply:
Thanks for the heads-up @dmfenton, I have revised the package number and fixed the code style issue. Let me know if you spot anything else.